ops/deployDaoStack.js

```diff
@@ -0,0 +1,46 @@
+const DAOstackMigration = require('@daostack/migration');
```

Comment:
this script replaces `ops/migration.js` and uses the scripts from the mgration repo instead